src/test/java/com/wikia/webdriver/common/remote/operations/CreateCategory.java

```diff
@@ -13,27 +15,31 @@
 public class CreateCategory {
 
   private static final int PARENT_ID = 1;
-
   public static final String CREATE_CATEGORY_URL_SUFFIX = "%s/forums";
-
   private final PostRemoteOperation remoteOperation;
 
   CreateCategory(User user) {
     remoteOperation = new PostRemoteOperation(user);
   }
 
-  public CategoryPill.Data execute(final CreateCategoryContext context) {
+  public CategoryPill.Data execute(final CreateCategoryContext context) throws RuntimeException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'java.lang.RuntimeException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)